### DISCUSSION TITLE
How can I change the axis color?

### DISCUSSION THREAD

**Discussion by StefanyHF** [patrykandpatrick/vico#629] [Questions] [answered]
I am currently using a dark background, and due to the default black color of the line chart axis, it is not visible.

**Comment by EnrikasVai** [comment, marked as answer]
Hello, in Composable private fun ComposeChart
```
startAxis = rememberStartAxis(
                label = rememberAxisLabelComponent(Color.White),
                axis = rememberAxisLineComponent(Color.White),
                guideline = rememberAxisGuidelineComponent(Color.White)
            ),
            bottomAxis = rememberBottomAxis(
                label = rememberAxisLabelComponent(Color.White),
                axis = rememberAxisLineComponent(Color.White),
                guideline = rememberAxisGuidelineComponent(Color.White),
                tickLength = 10.dp,
                valueFormatter = axisFormatter,
                ),
```

**Reply by StefanyHF** [reply]
thanks for your answer. I couldn't find rememberAxisLabelComponent and the others as well, do I need another import? 
Also, I have a lineChart that one of the parameters is axisValuesOverrider. Do you know if these changes should be passed as arguments to this axisValuesOverrider?

**Reply by patrickmichalik** [reply]
Hello! It sounds like you’re using Vico 1, @StefanyHF. The same functions are available, but their names aren’t prefixed with `remember`. Regarding `AxisValuesOverrider`, there’s no need to use it here. Thanks for the help, @EnrikasVai!

**Reply by StefanyHF** [reply]
Hello @patrickmichalik! I've now updated to Vico 2, and was able to apply what @EnrikasVai suggested, and it worked. Thank you both :)

**Reply by patrickmichalik** [reply]
I’m glad to hear that. You’re welcome!
